src/ui/SearchView.java

```diff
@@ -156,5 +152,74 @@ public void performSearch(String zipCode) {
     public JComponent $$$getRootComponent$$$() {
         return basePane;
     }
+
+    /**
```

Comment:
I moved this back in because I thought we would be able to use the same renderer for both my VisitedList and the SearchView, but Im not really using a "user renderer" as I pass in a visit object for mine. Theres a bit of duplicated code with the image creation part, but other than that I think they should be separate.